content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -42,35 +42,71 @@ Istio control plane, Envoy can then communicate with the **single**
 Istio control plane and form a mesh network across multiple Kubernetes
 clusters.
 
-## Deploy all Kubernetes clusters to be used in the mesh
-
-After deployment of remote clusters, each one will have a
-credentials file associated with the admin context typically
-located in `$HOME/.kube/config`.
+## Create service account in remote clusters
```

Comment:
Consider the title "Create a kubeconfig file for each remote cluster"

The end result at the end of this section is a kubeconfig file that grants read access to services, endpoints, and pods, nodes in a remote cluster.

Reply:
Yeah, I tried to make that clear in the first paragraph about the kubeconfig but maybe your point about putting reference to kubeconfig in the section title is more clear.  Do you think a bulleted summary of what the procedure is doing would be easier to read?

Reply:
That sounds good. So the purpose is to create a kubeconfig access file that grants read access to those objects in a remote cluster, which is going to loaded into a secret object with a multi cluster label in the primary cluster. A bulleted summary will help people understand better with the workflow.